File: us/zonix/ctf/utils/ArmorUtil.java
package us.zonix.ctf.utils;

import org.bukkit.Color;
import org.bukkit.Material;
import org.bukkit.inventory.ItemStack;
import org.bukkit.inventory.meta.LeatherArmorMeta;

public class ArmorUtil {

    public static ItemStack generate(Material mat, Color color) {
        ItemStack armor = new ItemStack(mat);
        LeatherArmorMeta meta4 = (LeatherArmorMeta) armor.getItemMeta();
        meta4.setColor(color);
        armor.setItemMeta(meta4);
        return armor;
    }

}

File: us/zonix/ctf/utils/ArmorUtil.kt
package us.zonix.ctf.utils

import org.bukkit.Color
import org.bukkit.Material
import org.bukkit.inventory.ItemStack
import org.bukkit.inventory.meta.LeatherArmorMeta

object ArmorUtil {

    fun generate(mat: Material?, color: Color?): ItemStack {
        val armor = ItemStack(mat)
        val meta4 = armor.itemMeta as LeatherArmorMeta
        meta4.color = color
        armor.itemMeta = meta4
        return armor
    }
}